sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerSessionPage.scala

```diff
@@ -147,42 +147,6 @@ private[ui] class ThriftServerSessionPage(parent: ThriftServerTab)
     <td>{errorSummary}{details}</td>
   }
 
-  /** Generate stats of batch sessions of the thrift server program */
-  private def generateSessionStatsTable(): Seq[Node] = {
```

Comment:
![1](https://user-images.githubusercontent.com/26266482/26870645-37a00c12-4ba3-11e7-9633-3624a920168f.png)

@srowen
This method implements the functions shown in the figure